packages/core/src/external/athenahealth/index.ts

```diff
@@ -397,6 +437,238 @@ class AthenaHealthApi {
+    );
+    const chartProblemUrl = `/chart/${this.stripPatientId(patientId)}/problems`;
+    try {
+      const snomedCode = this.getSnomedConditionCode(condition);
+      if (!snomedCode) {
+        throw new MetriportError("No SNOMED code found for condition", undefined, {
+          cxId,
+          practiceId: this.practiceId,
+          patientId,
+          departmentId,
+          conditionId: condition.id,
+        });
+      }
+      const startDate = this.getConditionStartDate(condition);
+      if (!startDate) {
+        throw new MetriportError("No SNOMED code found for condition", undefined, {
+          cxId,
+          practiceId: this.practiceId,
+          patientId,
+          departmentId,
+          conditionId: condition.id,
+        });
+      }
+      const conditionStatus = this.getConditionStatus(condition);
+      const athenaStatus = conditionStatus
+        ? problemStatusesAthena.find(
+            status => status.toLowerCase() === conditionStatus.toLowerCase()
+          )
+        : undefined;
+      const data = {
+        departmentid: this.stripDepartmentId(departmentId),
+        note: "Added via Metriport App",
+        snomedcode: snomedCode,
+        startdate: this.formatDate(startDate),
+        status: athenaStatus,
+        THIRDPARTYUSERNAME: undefined,
+        PATIENTFACINGCALL: undefined,
+      };
+      const response = await this.axiosInstanceProprietary.post(
+        chartProblemUrl,
+        this.createDataParams(data)
+      );
+      if (!response.data) throw new MetriportError(`No body returned from ${chartProblemUrl}`);
+      debug(`${chartProblemUrl} resp: `, () => JSON.stringify(response.data));
+      if (responsesBucket) {
+        const filePath = createHivePartitionFilePath({
+          cxId,
+          patientId,
+          date: new Date(),
+        });
+        const key = `athenahealth/chart/problem/${filePath}/${uuidv7()}.json`;
+        this.s3Utils
+          .uploadFile({
+            bucket: responsesBucket,
+            key,
+            file: Buffer.from(JSON.stringify(response.data), "utf8"),
+            contentType: "application/json",
+          })
+          .catch(processAsyncError("Error saving to s3 @ AthenaHealth - createProblem"));
+      }
+      const outcome = problemCreateResponseSchema.parse(response.data);
+      if (!outcome.success) throw new MetriportError(`Problem create not successful`);
+      return outcome;
+    } catch (error) {
+      const msg = `Failure while creating problem @ AthenaHealth`;
+      log(`${msg}. Cause: ${errorToString(error)}`);
+      capture.error(msg, {
+        extra: {
+          url: chartProblemUrl,
+          cxId,
+          practiceId: this.practiceId,
+          patientId,
+          departmentId,
+          context: "athenahealth.create-problem",
+          error,
+        },
+      });
+      throw error;
```

Comment:
I think this will result in two events sent to Sentry.

Could throw `MetriportError`, passing `error` as the second parameter and `extra` above (minus `error`) on `additionalInfo`.

Reply:
there's only one capture in the method.. do you mean this an the route level captures?

Reply:
Will leave as is for now.